Example/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>AppCenterSecret</key>
+	<string>$(RMA_APP_CENTER_SECRET)</string>
```

Comment:
Did you also add this secret to the `prod` build job on App Center?

Reply:
yes it is :)